### ISSUE TITLE
ReflectionException: Class Swift_AddressEncoder_IdnAddressEncoder does not exist

### ISSUE THREAD

**Issue by josteph** [swiftmailer/swiftmailer#1114]
| Q                   | A
| ------------------- | -----
| Bug report?         | yes
| Feature request?    | no
| RFC?                | no
| How used?           | Laravel v5.6.28
| Swiftmailer version | 6.1.2
| PHP version         | 7.1.7

### Observed behaviour
```
{
    "message": "Class Swift_AddressEncoder_IdnAddressEncoder does not exist",
    "exception": "ReflectionException",
    "file": "/Users/agus/Sites/careerkuweb/vendor/swiftmailer/swiftmailer/lib/classes/Swift/DependencyContainer.php",
    "line": 327,
    "trace": [
        {
            "file": "/Users/agus/Sites/careerkuweb/vendor/swiftmailer/swiftmailer/lib/classes/Swift/DependencyContainer.php",
            "line": 327,
            "function": "__construct",
            "class": "ReflectionClass",
            "type": "->"
        },
        {
            "file": "/Users/agus/Sites/careerkuweb/vendor/swiftmailer/swiftmailer/lib/classes/Swift/DependencyContainer.php",
            "line": 115,
            "function": "createNewInstance",
            "class": "Swift_DependencyContainer",
            "type": "->"
        },
        {
            "file": "/Users/agus/Sites/careerkuweb/vendor/swiftmailer/swiftmailer/lib/classes/Swift/DependencyContainer.php",
            "line": 389,
            "function": "lookup",
            "class": "Swift_DependencyContainer",
            "type": "->"
        },
        {
            "file": "/Users/agus/Sites/careerkuweb/vendor/swiftmailer/swiftmailer/lib/classes/Swift/DependencyContainer.php",
            "line": 366,
            "function": "lookupRecursive",
            "class": "Swift_DependencyContainer",
            "type": "->"
        },
        {
            "file": "/Users/agus/Sites/careerkuweb/vendor/swiftmailer/swiftmailer/lib/classes/Swift/DependencyContainer.php",
            "line": 134,
            "function": "resolveArgs",
            "class": "Swift_DependencyContainer",
            "type": "->"
        },
        {
            "file": "/Users/agus/Sites/careerkuweb/vendor/swiftmailer/swiftmailer/lib/classes/Swift/SmtpTransport.php",
            "line": 37,
            "function": "createDependenciesFor",
            "class": "Swift_DependencyContainer",
            "type": "->"
        },
        {
            "file": "/Users/agus/Sites/careerkuweb/vendor/laravel/framework/src/Illuminate/Mail/TransportManager.php",
            "line": 34,
            "function": "__construct",
            "class": "Swift_SmtpTransport",
            "type": "->"
        },
        {
            "file": "/Users/agus/Sites/careerkuweb/vendor/laravel/framework/src/Illuminate/Support/Manager.php",
            "line": 96,
            "function": "createSmtpDriver",
            "class": "Illuminate\\Mail\\TransportManager",
            "type": "->"
        },
        {
            "file": "/Users/agus/Sites/careerkuweb/vendor/laravel/framework/src/Illuminate/Support/Manager.php",
            "line": 71,
            "function": "createDriver",
            "class": "Illuminate\\Support\\Manager",
            "type": "->"
        },
        {
            "file": "/Users/agus/Sites/careerkuweb/vendor/laravel/framework/src/Illuminate/Mail/MailServiceProvider.php",
            "line": 102,
            "function": "driver",
            "class": "Illuminate\\Support\\Manager",
            "type": "->"
        },
        {
            "file": "/Users/agus/Sites/careerkuweb/vendor/laravel/framework/src/Illuminate/Container/Container.php",
            "line": 764,
            "function": "Illuminate\\Mail\\{closure}",
            "class": "Illuminate\\Mail\\MailServiceProvider",
            "type": "->"
        },
        ...
    ]
}
```
The class dependency still being used in those files:
```
Searching 150 files for "Swift_AddressEncoder_IdnAddressEncoder"

/Users/agus/Sites/careerkuweb/vendor/swiftmailer/swiftmailer/lib/classes/Swift/Mime/Headers/IdentificationHeader.php:
   46          $this->setFieldName($name);
   47          $this->emailValidator = $emailValidator;
   48:         $this->addressEncoder = $addressEncoder ?? new Swift_AddressEncoder_IdnAddressEncoder();
   49      }
   50  

/Users/agus/Sites/careerkuweb/vendor/swiftmailer/swiftmailer/lib/classes/Swift/Mime/Headers/MailboxHeader.php:
   45          $this->setEncoder($encoder);
   46          $this->emailValidator = $emailValidator;
   47:         $this->addressEncoder = $addressEncoder ?? new Swift_AddressEncoder_IdnAddressEncoder();
   48      }
   49  

/Users/agus/Sites/careerkuweb/vendor/swiftmailer/swiftmailer/lib/classes/Swift/Mime/Headers/PathHeader.php:
   44          $this->setFieldName($name);
   45          $this->emailValidator = $emailValidator;
   46:         $this->addressEncoder = $addressEncoder ?? new Swift_AddressEncoder_IdnAddressEncoder();
   47      }
   48  

/Users/agus/Sites/careerkuweb/vendor/swiftmailer/swiftmailer/lib/classes/Swift/Mime/SimpleHeaderFactory.php:
   41          $this->emailValidator = $emailValidator;
   42          $this->charset = $charset;
   43:         $this->addressEncoder = $addressEncoder ?? new Swift_AddressEncoder_IdnAddressEncoder();
   44      }
   45  

/Users/agus/Sites/careerkuweb/vendor/swiftmailer/swiftmailer/lib/classes/Swift/Transport/AbstractSmtpTransport.php:
   51          $this->buffer = $buf;
   52          $this->eventDispatcher = $dispatcher;
   53:         $this->addressEncoder = $addressEncoder ?? new Swift_AddressEncoder_IdnAddressEncoder();
   54          $this->setLocalDomain($localDomain);
   55      }

5 matches across 5 files
```

### Expected behaviour
It should be fixed here #1100, but I encountered it just now.

### Example
Send test email to email_something@email.com (with underscore).
EDIT: Apparently, encountered this in any email tested. Probably with installation or upgrading issue. Any solution here?

**Comment by JonMoreno** [comment]
# There is bug with swiftmailer.
## If you wish to make a quick fix simply lock down 6.0 version of swiftmailer in your project composer.json
{
    "require": {
        "laravel/framework": "5.6.*",
        "swiftmailer/swiftmailer": "6.0.*",
         "...": "..."
     },
     "...": "..."
}

**Comment by fabpot** [comment]
There is no bug in Swiftmailer, what is you issue?
